lldb/examples/python/crashlog.py

```diff
@@ -526,6 +526,47 @@ def create_target(self):
     def get_target(self):
         return self.target
 
+    def load_images(self, options, loaded_images=[]):
```

Comment:
Suggestion: Default `loaded_images` to `None`. You probably don't want `loaded_images` to default to `[]`. It will persist between calls, e.g.:
```
>>> def foo(x, lst=[]):
...     lst.append(x)
...     print(lst)
...
>>> foo(1)
[1]
>>> foo(2)
[1, 2]
```

Reply:
Good point